### PR TITLE
Scroll event list on Window Back

### PR DESCRIPTION
**Pull request by martypowell**
I wasn't sure how to get this to work other than wrapping in a timeout. It seems like there is some conflict with the scroll to top logic that causes the page to scroll again on the list.

### CODE REVIEW

File: js/apps/events-page/controllers/eventsPageCtrl.js [hunk 1]
```diff
@@ -14,6 +14,10 @@
 		metaService,
 		RequestModel
 	) {
+        setTimeout(() => {
```
Comment (mxsnyder) [inline js/apps/events-page/controllers/eventsPageCtrl.js]: You probably already tried it, but would $document.ready() work here?

Comment (mxsnyder) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: Also, does the "pop" the window into place? You could always trigger the scrollToTop button instead, which is a smooth scrolling effect.

Comment (mxsnyder) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: Adding a stupid comment so I can stupid request changes.

Comment (martypowell) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: i wonder if window.onload might work here

Comment (martypowell) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: I tried all of these without success. Any other thoughts?

Comment (mxsnyder) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: I guess we're going with setTimeout then. Worse things have happened.

Comment (martypowell) [reply js/apps/events-page/controllers/eventsPageCtrl.js]: ☹️

**Pull request merged**